yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -279,6 +279,22 @@ const (
 		WHERE read_write_rates.schema_name = table_index_stats.schema_name
 		  AND read_write_rates.object_name = table_index_stats.object_name
 	);`
+
+	/*
+		changing to TABLE type to make the object_type field consistent across Oracle and PG
```

Comment:
as discussed, pls revert this as it is not required. This detail might be useful to sizing calculator somewhere down the line